init.lua

```diff
@@ -13,6 +13,9 @@ jumpdrive = {
 
 		-- base technic power requirement
 		powerrequirement = tonumber(minetest.settings:get("jumpdrive.power_requirement")) or 2500,
+
+		-- allow emerging area on "uncharted" error
+		emerge_uncharted = core.settings:get_bool("allow_jumpdrive_emerge", false),

```

Comment:
```suggestion
		emerge_uncharted = core.settings:get_bool("jumpdrive.allow_emerge", false),
```
To follow naming style of other jumpdrive settings (comment from previous round).

Reply:
should be done